ftl/common/args.py

```diff
@@ -102,6 +102,12 @@ def base_parser():
         action='store',
         default=constants.DEFAULT_ENTRYPOINT,
         help='The entrypoint for the dockerimage')
+    parser.add_argument(
```

Comment:
Why not just do this all the time instead of as a flag?

Reply:
a flag was requested by the python team, I originally did add it all the time but then it was mentioned that they might want to do it as a flag option.  They wanted this so that they could use env variables in their args